cnvlib/coverage.py

```diff
@@ -104,15 +112,61 @@ def filter_read(read):
     return count, depth
 
 
-def interval_coverages_pileup(bed_fname, bam_fname, min_mapq):
+def rm(path):
+    try:
+        os.unlink(path)
+    except OSError:
+        pass
+
+def to_chunks(bed_fname, chunk_size=5000):
```

Comment:
The chunk size could be calculated on the fly from the number of processes and lines in the BED file. The BED file can turn into a GenomicArray instance with some bedtools-like features via `tabio.read(bed_fname, "bed")`.